src/nvim/fold.c

```diff
@@ -274,14 +274,11 @@ int foldLevel(linenr_T lnum)
   return foldLevelWin(curwin, lnum);
 }
 
-/* lineFolded()	{{{2 */
-/*
- * Low level function to check if a line is folded.  Doesn't use any caching.
- * Return TRUE if line is folded.
- * Return FALSE if line is not folded.
- * Return MAYBE if the line is folded when next to a folded line.
- */
-int lineFolded(win_T *win, linenr_T lnum)
+// lineFolded() {{{2
+// Low level function to check if a line is folded.  Doesn't use any caching.
+// Return true if line is folded.
+// Return false if line is not folded.
+bool lineFolded(win_T *const win, const linenr_T lnum)
 {
   return foldedCount(win, lnum, NULL) != 0;
```

Comment:
This expression looks boolean to me and doesn't return `MAYBE` so i changed this bool and updated affected code.